### PR TITLE
Fix: CREATE2 opcode

### PR DESCRIPTION
**Pull request by ArthurBraud**

### CODE REVIEW

File: docs/opcodes/F5.mdx [hunk 1]
```diff
@@ -5,17 +5,22 @@ group: System operations
 
 ## Notes
 
-This instruction has no effect on the EVM state. See [here](https://ethereum.org/en/developers/docs/smart-contracts/anatomy/#events-and-logs).
+Equivalent to [CREATE](/#F0), except the salt allows to create at a deterministic address.
```
Comment (assimovt) [inline docs/opcodes/F5.mdx:8]: typo: `create` -> `to be created` ?

**Review by assimovt — CHANGES_REQUESTED** [review]

File: docs/opcodes/F5.mdx (hunk 1 above)
Comment (ArthurBraud) [reply docs/opcodes/F5.mdx]: +1 I changed it

File: docs/opcodes/F5.mdx [hunk 2]
```diff
@@ -5,17 +5,22 @@ group: System operations
 
 ## Notes
 
-This instruction has no effect on the EVM state. See [here](https://ethereum.org/en/developers/docs/smart-contracts/anatomy/#events-and-logs).
+Equivalent to [CREATE](/#F0), except the salt allows to be created at a deterministic address.
```
Comment (peroket) [inline docs/opcodes/F5.mdx:8]: I realise the sentence is not complete: "the salt allows *the new contract* to be created

**Review by peroket — DISMISSED** [review]

File: docs/opcodes/F5.mdx (hunk 2 above)
Comment (ArthurBraud) [reply docs/opcodes/F5.mdx]: +1 I changed it

**Comment by ArthurBraud** [comment]
ptal @peroket

**Review by peroket — APPROVED** [review]

**Review by assimovt — APPROVED** [review]

**Pull request merged**